saleor/graphql/account/mutations.py

```diff
@@ -545,12 +547,14 @@ def perform_mutation(cls, root, info, address_id, user_id, type):
             raise ValidationError({
                 'address_id': 'The address doesn\'t belong to that user.'})
 
-        if type == AddressTypeEnum.BILLING.value:
+        raw_address_type = data.get('type')
```

Comment:
I would stick with `address_type` name, `raw` does not give (for me) any useful information.